haskeline.cabal

```diff
@@ -20,7 +20,7 @@ Homepage:       http://trac.haskell.org/haskeline
 Bug-Reports:    https://github.com/judah/haskeline/issues
 Stability:      Stable
 Build-Type:     Custom
-extra-source-files: examples/Test.hs Changelog
+extra-source-files: examples/Test.hs Changelog includes/*.h
```

Comment:
Thanks for the explanation.  My guess is it was a mistake to put `win_console.h` in `install-includes`, and it should also be listed in `extra-source-files` instead.  But OK to leave as-is for now.